src/applications/gi/utils/render.jsx

```diff
@@ -3,24 +3,45 @@ import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
 
 export const renderSchoolClosingAlert = result => {
   const { schoolClosing } = result;
+
   if (!schoolClosing) return null;
   return (
     <AlertBox
-      content={<p>Upcoming campus closure</p>}
-      headline="School closure"
+      content={<li>Upcoming campus closure</li>}
+      headline="A campus is closing soon"
       isVisible={!!schoolClosing}
       status="warning"
     />
   );
 };
 
+const renderReasons = cautionFlags => {
+  const flags = [];
+  cautionFlags
+    .sort((a, b) => {
+      if (a.reason.toLowerCase() < b.reason.toLowerCase()) return -1;
+      if (a.reason.toLowerCase() > b.reason.toLowerCase()) return 1;
+      return 0;
+    })
+    .forEach(flag => {
+      flags.push(<li key={flag.id}>{flag.reason}</li>);
+    });
+
+  return flags;
+};
+
 export const renderCautionAlert = result => {
-  const { cautionFlag } = result;
+  const { cautionFlag, cautionFlags } = result;
+
   if (!cautionFlag) return null;
```

Comment:
why not check if `cautionFlags` is empty and remove use of this attribute?
Since in https://github.com/department-of-veterans-affairs/gibct-data-service/pull/612 this attribute just becomes an empty check against the array and eventually we might want to remove `caution_flag` and `caution_flag_reason` as attributes being returned from GIDs

Reply:
I had thought about doing that actually, that way we dont have to pass it over either. I'll go ahead and make that switch.